### DISCUSSION TITLE
found a working apk source

### DISCUSSION THREAD

**Discussion by trustytrojan** [vinegarhq/sober#113] [Show and tell]
just want to share with others because it worked for me
https://www.apkmirror.com/apk/roblox-corporation/roblox/roblox-2-638-610-release/roblox-2-638-610-android-apk-download/

**Comment by Amgsdfyhadfgybi** [comment]
u cant share stuff like this i think

**Comment by Amgsdfyhadfgybi** [comment]
yeah no i dont think u can do this, i suggest just deleting it before it causes something

**Comment by CyberKite** [comment]
just get the apks from Google, for your own safety. There are tools to get Google CDN links

**Comment by trustytrojan** [comment]
well nothing happened and now sober auto-downloads it for you anyway LMAO

**Reply by CyberKite** [reply]
how can you be so sure? Tell me, would you really notice a stray file in ~/.config? Or a stray entry in your crond, pointing to ~/.local/bin/aa-healthcheck, which is a shellscript curling a remote server?

**Reply by tunis4** [reply]
sober checks the signature of the apk to make sure its legit, so you dont have to worry about the apk being malicious
